pallets/order-book/src/lib.rs

```diff
@@ -414,18 +428,100 @@ pub mod pallet {
         pub fn update_orderbook(
             origin: OriginFor<T>,
             order_book_id: OrderBookId<AssetIdOf<T>>,
-            _tick_size: OrderPrice,
-            _step_lot_size: OrderVolume,
-            _min_lot_size: OrderVolume,
-            _max_lot_size: OrderVolume,
+            tick_size: OrderPrice,
+            step_lot_size: OrderVolume,
+            min_lot_size: OrderVolume,
+            max_lot_size: OrderVolume,
         ) -> DispatchResult {
             ensure_root(origin)?;
```

Comment:
https://github.com/sora-xor/sora2-network/pull/468#pullrequestreview-1448896560

Reply:
We will do that in the separate PR for task https://github.com/sora-xor/sora2-network/issues/400